server/core/src/irodsReServer.cpp

```diff
@@ -1,35 +1,36 @@
-#include "connection_pool.hpp"
+#include "catalog.hpp"
 #include "client_connection.hpp"
+#include "connection_pool.hpp"
 #include "initServer.hpp"
+#include "irodsReServer.hpp"
```

Comment:
consider moving this header file to the top and adding a new line after it.

that helps to signal that this cpp file is the implementation of irodsReServer.hpp and everything else is a dependency.

Reply:
Will do